packages/@sanity/field/src/diff/components/FieldChange.tsx

```diff
@@ -114,4 +116,44 @@ export function FieldChange({
       </FieldWrapper>
     </Stack>
   )
+
+  const hidden = change.schemaType.hidden
+
```

Comment:
Should we add the `checkProperty` below here: `const readOnly = readOnly || change?.parentSchema?.readOnly || change.schemaType.readOnly`

..and then use it below when doing the type check + passing it to `ConditionalPropertyResolver`?

Reply:
yeah, already on it 👍